x-pack/plugins/security_solution/public/alerts/components/alerts_table/index.tsx

```diff
@@ -157,21 +152,32 @@ export const AlertsTableComponent: React.FC<AlertsTableComponentProps> = ({
   );
 
   const onAlertStatusUpdateSuccess = useCallback(
-    (count: number, status: string) => {
-      const title =
-        status === 'closed'
-          ? i18n.CLOSED_ALERT_SUCCESS_TOAST(count)
-          : i18n.OPENED_ALERT_SUCCESS_TOAST(count);
-
+    (count: number, status: Status) => {
+      let title;
```

Comment:
Very small nit: I usually type these so they don't start out their scope as an `any` such as:
```ts
let title: string;
```

The side effect is as below, they start out as `any` and then TypeScript slowly derives their type but technically they could become any type not just string when its just `let title`;

<img width="558" alt="Screen Shot 2020-06-12 at 10 16 05 AM" src="https://user-images.githubusercontent.com/1151048/84523640-e167c400-ac95-11ea-85cd-258dade1841e.png">

<img width="547" alt="Screen Shot 2020-06-12 at 10 16 10 AM" src="https://user-images.githubusercontent.com/1151048/84523703-00feec80-ac96-11ea-8e06-e164b10ecb3c.png">